dsa-23au/java-dsa/arrays-links/src/main/java/dev/codewithfriends/ArrayWrapper.java

```diff
@@ -66,17 +78,31 @@ public void clear() {
 
     @Override
     public Object get(int index) {
-        return null;
+        return this._array[index];
     }
 
     @Override
     public Object set(int index, Object element) {
-        return null;
+        if (index > maxSize) {
+            return null;
+        }
+        else {
+            Object previousElement = this._array[index];
+            this._array[index] = element;
+            return previousElement;
```

Comment:
Nice, choosing to return the replaced element makes a lot of sense, as opposed to returning the new element.